css/style.css

```diff
@@ -57,6 +57,27 @@ a {
   text-decoration: none;
 }
 
+input {
+  line-height: 40px;
+  width: 300px;
```

Comment:
for buttons at least, it may be better to set a `min-width` so that way if the text inside the button is long it can continue to grow in width.

you can target buttons like `submit` separate from regular inputs by using the attribute selector:

```
input[type="submit"], 
input[type="button"] {
 /* styles specific to buttons go here */
}
```